ci-operator/jobs/openshift/machine-api-operator/openshift-machine-api-operator-master-presubmits.yaml

```diff
@@ -829,6 +829,76 @@ presubmits:
         secret:
           secretName: regcred
     trigger: (?m)^/test( | .* )e2e-openstack,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    cluster: api.ci
+    context: ci/prow/e2e-vsphere
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-machine-api-operator-master-e2e-vsphere
+    optional: true
+    rerun_command: /test e2e-vsphere
```

Comment:
This needs to be e2e-vsphere-operator like we do for any other provider running our machine API suite.

We use `e2e-providerX` for the origin suite over the installer